soc/arm64/renesas_rcar/gen3/mmu_regions.c

```diff
@@ -26,11 +23,12 @@ static const struct arm_mmu_region mmu_regions[] = {
 			      DT_REG_SIZE(DT_INST(0, renesas_r8a7795_cpg_mssr)),
 			      MT_DEVICE_nGnRnE | MT_RW | MT_NS),
 
-#if DT_NODE_HAS_STATUS(DT_INST(0, renesas_rcar_pfc), okay)
```

Comment:
I believe that this change should belong to previous commit.

Reply:
As I said, this change should be squashed to the previous commit

Reply:
done